bls/algebra/src/lib.rs

```diff
@@ -244,6 +241,56 @@ pub extern "C" fn hash_composite(
     })
 }
 
+#[no_mangle]
+pub extern "C" fn compress_signature(
+    in_signature: *const u8,
+    in_signature_len: c_int,
+    out_signature: *mut *mut u8,
+    out_len: *mut c_int,
+) -> bool {
+    convert_result_to_bool::<_, Box<dyn Error>, _>(|| {
+        let signature = unsafe { slice::from_raw_parts(in_signature, in_signature_len as usize) };
+        let x = Fq::read(&signature[0..48]).unwrap();
```

Comment:
Feels like the wrong level of abstraction to put this in `bls`. I prefer a magic number here over including a constant that might conflict with the `zexe` implementation itself, given we only need it once in `lib.rs`. Limits the debugging to the function itself in case something goes wrong. Ideally I think there would just be an accessible constant in the 377 field implementations.